definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -141,7 +141,6 @@ declare class moment$Moment {
   static utc(moment: moment$Moment): moment$Moment;
   static utc(date: Date): moment$Moment;
   static parseZone(rawDate: string): moment$Moment;
-  parseZone(): moment$Moment;
```

Comment:
The v2.3.x types never had this invalid instance method type:

https://github.com/flow-typed/flow-typed/blob/78c72eef4472905c8248ceeb51842dc912c5435a/definitions/npm/moment_v2.3.x/flow_v0.25.x-/moment_v2.3.x.js#L142-L160